sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -25,6 +25,7 @@ export class LroEngine<TResult, TState extends PollOperationState<TResult>> exte
 
 // @public
 export interface LroEngineOptions<TResult, TState> {
+    cancel?: (state: TState) => Promise<void>;
```

Comment:
would it make sense to pull this change out into its own PR that we can quickly merge to keep this review more focused on the TA changes?

Reply:
I duplicated it in https://github.com/Azure/azure-sdk-for-js/pull/21793. I will rebase the TA changes branch once the core-lro PR is merged.